cpanfile

```diff
@@ -0,0 +1,10 @@
+# Force an up-to-date version so TidyAll dependencies can be installed and
+# tested. Older versions of Storable do not handle Regexes.
+requires 'Storable', '>= 3.06';
```

Comment:
I'm wondering why it builds fine on Leap 15.4, though: https://build.opensuse.org/package/show/devel:languages:perl/perl-Log-Any

Reply:
> I'm wondering why it builds fine on Leap 15.4 and 15.5, though: 

Ah, because the failing test requires Devel::StackTrace, which is only a soft dependency.
However, we should keep in mind, if we require Storable >= 3.06, then we can't use openSUSE packages for that, because we only have Storable in the perl code in openSUSE, not the dual-life module.

In the cpanfile it's ok, but we couldn't have that in our spec file.
I'm wondering if the Log::Any test should actually be fixed to check for the Storable version, otherwise skip the test.
Would you want to make a PR for Log::Any? :)

Reply:
I made a quick PR: https://github.com/preaction/Log-Any/pull/105